src/content/app/tools/blast/hooks/useBlastForm.ts

```diff
@@ -0,0 +1,184 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useAppSelector, useAppDispatch } from 'src/store';
+
+import { useBlastConfigQuery } from 'src/content/app/tools/blast/state/blast-api/blastApiSlice';
+
+import {
+  setSequences as setSequencesAction,
+  setSequenceForGenome as setSequenceForGenomeAction,
+  setBlastProgram as setBlastProgramAction,
+  setSequenceType as setSequenceTypeAction,
+  setBlastDatabase as setBlastDatabaseAction,
+  updateEmptyInputDisplay as updateEmptyInputDisplayAction,
+  changeSensitivityPresets as changeSensitivityPresetsAction,
+  setHasUncommittedSequence as setHasUncommittedSequenceAction,
+  clearBlastForm as clearBlastFormAction,
+  type Species
+} from 'src/content/app/tools/blast/state/blast-form/blastFormSlice';
```

Comment:
Why is it necessary to distinguish these functions as actions?

Reply:
It isn't necessary. I am trying to avoid the naming clash with the functions declared inside the useBlastForm hook, and renaming the imports seemed like an easy way of doing so.